pset-iterators.js

```diff
@@ -43,6 +43,12 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+const firstWhigPresident = presidents.find(function(parties) {
+  return parties.party === "Whig"
+})
+
+console.log(firstWhigPresident.president)
+
```

Comment:
@rlou88 Great work here using the `find` method to return the first president in the whig party.

Take a look at this solution [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L35)